src/dotnet/CommonLocalizableStrings.cs

```diff
@@ -185,5 +185,16 @@ internal class CommonLocalizableStrings
         public const string ProjectNotCompatibleWithFrameworks = "Project `{0}` cannot be added due to incompatible targeted frameworks between the two projects. Please review the project you are trying to add and verify that is compatible with the following targets:";
         public const string ProjectDoesNotTargetFramework = "Project `{0}` does not target framework `{1}`.";
         public const string ProjectCouldNotBeEvaluated = "Project `{0}` could not be evaluated. Evaluation failed with following error:\n{1}";
+
+        /// common options
```

Comment:
If you look at the whole fine, it is organized by comment and sections. I think it is ok in this case. In general I try to reduce comments as much as I could, however, localization.cs file is more close to setting than actual code.